tests/commands/record/test_build.py

```diff
@@ -1,25 +1,14 @@
 import json
 import os
-import shutil
-import tempfile
 from unittest import mock
 
 import responses  # type: ignore
 
-from launchable.utils.session import SESSION_DIR_KEY, clean_session_files, read_build
+from launchable.utils.session import read_build
 from tests.cli_test_case import CliTestCase
 
 
 class BuildTest(CliTestCase):
-    def setUp(self):
```

Comment:
It's already defined in CliTestCase.

https://github.com/launchableinc/cli/blob/7c61265e748f4551e937a295878c427255931434/tests/cli_test_case.py#L31-L33